src/Filters/FilterFunction.js

```diff
@@ -0,0 +1,33 @@
+const sortby =({sortBy},array)=>{
+    switch(sortBy){
+        case "HIGHTOLOW":
+            return sortBy==="HIGHTOLOW"?[...array].sort((a,b)=>Number(b.price)- Number(a.price)):[...array];
+        case "LOWTOHIGH":
+            return sortBy==="LOWTOHIGH"?[...array].sort((a,b)=>Number(a.price)- Number(b.price)):[...array];
+        default:
+            return [...array]
```

Comment:
Since you are not changing anything in default, you can return the array normally without spreading

Reply:
Changed it in the latest commit.